files/templates/volunteer_janitor.html

```diff
@@ -14,6 +14,8 @@
         <div>
             <div class="comment">
                 {% with comments=[c] %}
+                    {% set should_hide_username = false %}
```

Comment:
Perhaps it doesn't work because this is `false`?

Just need to make sure that this doesn't break behavior elsewhere when undefined.